cruise-control-metrics-reporter/src/test/java/com/linkedin/kafka/cruisecontrol/metricsreporter/utils/CCEmbeddedBroker.java

```diff
@@ -38,7 +38,7 @@ public CCEmbeddedBroker(Map<Object, Object> config) {
 
     try {
       // Also validates the config
-      KafkaConfig kafkaConfig = KafkaConfig.apply(config);
+      KafkaConfig kafkaConfig = KafkaConfig.apply(config, true);
```

Comment:
Would using `new KafkaConfig(config, true)` make this line compatible with pre- and post-2.8 Kafka dependencies?